tests/test_utility/test_hex_utils.py

```diff
@@ -49,6 +50,22 @@ def test_hex_to_utf8(self):
         utfstr = hex_to_utf8(binhex)
         self.assertEqual(utfstr, "")
 
+    def test_hex_to_byte_arrary(self):
```

Comment:
Add some negative cases including one mentioned in above comment.

Reply:
Done